npm/design-system/src/components/FileExplorer/FileExplorer.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { mount } from '@cypress/react'
+import React from 'react'
+import { FileExplorer } from './FileExplorer'
+import { FileLike } from './types'
+
+const files: FileLike[] = [
+  {
+    name: 'foo/bar/foo.spec.js',
+    onClick: (e, foo) => {
+    },
+    isOpen: false,
+  },
+  // @ts-ignore
+  { name: 'bar/foo.spec.tsx' },
+  // @ts-ignore
+  { name: 'merp/foo.spec.ts' },
+]
+
+describe('FileExplorer', () => {
+  it('renders', () => {
```

Comment:
Any assertions? At least cy.persySnapshot?

Reply:
No, I was using it as a playground, not for specs. It still needs specs... just was trying to get something up for the src to be reviewable.